docs/specifications/tests/Nameserver-TP/nameserver02.md

```diff
@@ -1,61 +1,113 @@
-## NAMESERVER02: Test of EDNS0 support
+# NAMESERVER02: Test of EDNS0 support
 
-### Test case identifier
-**NAMESERVER02** Test of EDNS0 support
+## Test case identifier
+**NAMESERVER02**
 
-### Objective
+## Objective
 
 EDNS(0) is a mechanism to announce capabilities of a DNS implementation,
 and is now basically required by any new functionality in DNS such as
-DNSSEC.
-Initially standardized in [RFC 2671](https://tools.ietf.org/search/rfc2671)
-and later updated by [RFC 6891](https://tools.ietf.org/search/rfc6891),
-EDNS(0) is a mechanism to announce capabilities of a DNS implementation.
+DNSSEC. EDNS(0) is standardized in [RFC 6891].
+
 This test case checks that all name servers has the capability to do
-EDNS(0).
+EDNS(0) or if not, correctly replies to queries containing EDNS
+(OPT record).
 
-Servers not implementing EDNS(0) returns FORMERR:
+Servers not supporting EDNS(0) must return FORMERR 
+([RFC 6891, section 7]):
 
 > Responders that choose not to implement the protocol extensions
 > defined in this document MUST respond with a return code (RCODE) of
 > FORMERR to messages containing an OPT record in the additional
 > section and MUST NOT include an OPT record in the response.
 
-### Inputs
-
-The domain name to be tested.
+Servers supporting EDNS(0) must reply with EDNS(0)
+([RFC 6891, section 6.1.1]):
```

Comment:
Add the following text: 
 "The EDNS    query should specify a UDP buffer size of 512 bytes to avoid false classification of not supporting EDNS due to response packet size."

Reply:
Updated. I do not know how much it matters in practice.